package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "alexa-openwhisk",
+  "name": "alexa",
```

Comment:
"alexa" is not very unique (and there is already a package of that name).  I'm not sure why this existing code patten needs a new name.